.github/workflows/release-binaries-all.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
-          - ubuntu-22.04
+          - ubuntu-24.04
```

Comment:
It's by design that we want to build release binaries on the oldest supported Ubuntu version, because they will work on newer versions too.  Binaries built on new versions of Ubuntu won't work on older versions.